setup.py

```diff
@@ -74,6 +77,19 @@ def clean_version(v):
       setup_requires=[],
       )
 
+if current_platform != "windows":
+    bash_file = "~/.bash_profile" if current_platform == "darwin" else "~/.bashrc"
```

Comment:
Thanks @henrykironde. If we are not on windows, we are either in OS X (darwin) or Linux (linux). The assignment of the bash_file would be "~/.bash_profile" if we are on darwin, and "~/.bashrc" otherwise, i.e. linux. Should I remove the assumption that "if not windows and darwin, the platform could only be linux?"